app/assets/stylesheets/components/_advert.scss

```diff
@@ -1,12 +1,17 @@
 .advert {
   @include clearfix();
   padding: 0 $advert-padding;
-  margin: $baseline-unit*6 0;
+  margin: 0 0 $baseline-unit*6 0;
   background-color: $advert-background-color;
+
+  @include respond-to($mq-m) {
+    margin-top: $baseline-unit*15.5;
+  }
 }
 
 .advert__intro {
   margin: $advert-padding 0;
+  color: $color-black;
```

Comment:
Really this should reference another colour variable, which references `$color-black`